src/gui/sdlmain.cpp

```diff
@@ -2872,6 +2872,10 @@ static void GUI_StartUp(Section *sec)
 	if (!SetDefaultWindowMode())
 		E_Exit("Could not initialize video: %s", SDL_GetError());
 
+	int transparency = section->Get_int("transparency");
+	double alpha = (double)(100 - transparency) / 100;
```

Comment:
could get a zero-divide here if `transparency` is 100.
suggest clamping between 0 and 90, then multiplying by `0.01`.

(something like)
`const auto alpha_percent = 0.01 * clamp(transparency, 0, 90);`